config/kbin_routes/admin.yaml

```diff
@@ -1,7 +1,25 @@
-admin_users:
-    controller: App\Controller\Admin\AdminUserController
+admin_users_active:
+    controller: App\Controller\Admin\AdminUsersActiveController
     defaults: { withFederated: false }
-    path: /admin/users/{withFederated}
+    path: /admin/users/active/{withFederated}
+    methods: [GET]
+
+admin_users_inactive:
+    controller: App\Controller\Admin\AdminUsersInactiveController
+    defaults: { withFederated: false }
+    path: /admin/users/inactive
+    methods: [GET]
+
+admin_users_suspended:
+    controller: App\Controller\Admin\AdminUsersSuspendedController
+    defaults: { withFederated: false }
+    path: /admin/users/suspended/{withFederated}
+    methods: [GET]
+
+admin_users_banned:
+    controller: App\Controller\Admin\AdminUsersBannedController
+    defaults: { withFederated: false }
+    path: /admin/users/banned/{withFederated}
```

Comment:
You could put all Controllers in a single file and just different methods. You can reference them via `App\Controller\Admin\AdminUsersBannedController::methodName`

Reply:
good suggestion!